package-lock.json

```diff
@@ -1,9736 +1,8 @@
 {
   "name": "junknet",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You're running an old version of `npm` and it's trying to downgrade the lockfile (more importantly, it doesn't understand the lockfile so you may not get the right versions).

    npm i -g npm

should update it.

Reply:
I was using my OS's version of `npm` (6.14.11), which didn't support lockfile v2s. 

New commit addresses this. I'm now using a locally built version, so this issue won't recur.